webapp/srpm_pkg_names.py

```diff
@@ -12,7 +12,7 @@ class SRPMPkgNamesAPI:
 
     def __init__(self, cache):
         self.cache = cache
-
+    @profile
```

Comment:
This line shouldn't be in production code.

Reply:
Yes, I forgot it.